cpp/log/cert.h

```diff
@@ -27,12 +28,11 @@ class Cert {
   // so caller should check IsLoaded() before doing anything else.
   // All attempts to operate on an unloaded cert will fail with ERROR.
   explicit Cert(const std::string& pem_string);
-  Cert() : x509_(NULL) {
+  Cert() : x509_() {
   }
-  ~Cert();
 
   bool IsLoaded() const {
-    return x509_ != NULL;
+    return x509_ != nullptr;
```

Comment:
Bleh, there's an `operator bool()`, but it's marked explicit. The compiler could tell you that! :stuck_out_tongue: 

I've used `.get()` for this before, but this is fine, if you're going to have a bit of fluff, one or the other, doesn't matter much...